### PR TITLE
[NU-1726] new quickstart

### PR DESCRIPTION
**Pull request by coutoPL**
In this PR:
- flat docker-compose (based on the installation example)
- setup container to configure schemas, topics, example scenarios
- example scenario data generators

----
Installing and running: `curl -o- https://raw.githubusercontent.com/TouK/nussknacker-quickstart/feature/NU-1726/new-quickstart/download-and-start.sh | bash`

or:

Running: `new-quickstart/start.sh`
Stop & clean: `new-quickstart/stop-and-clean.sh`

----
In the next PR I will replace the old quickstart with the new one and adapt documentation to the new one.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+      DB_URL: "jdbc:postgresql://postgres:5432/nu-db"
+      DB_DRIVER: "org.postgresql.Driver"
+      DB_USER: "nu"
+      DB_PASSWORD: "nupassword"
+      NUSSKNACKER_LOG_LEVEL: "INFO"
+      KAFKA_ADDRESS: "kafka:9092"
+      SCHEMA_REGISTRY_URL: "http://schema-registry:8081"
+      INFLUXDB_URL: "http://influxdb:8086"
+      FLINK_REST_URL: "http://flink-jobmanager:8081"
+      FLINK_QUERYABLE_STATE_PROXY_URL: "flink-taskmanager:9069"
```
Comment (raphaelsolarski) [inline docker-compose.yml:49]: IMO this one is no longer needed - we don't use queryable state anymore.

File: new-quickstart/run.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
```
Comment (raphaelsolarski) [inline new-quickstart/run.sh:1]: if this one and others bash scripts are run directly on host then please use:
```bash
#!/usr/bin/env bash
set -e
```

File: new-quickstart/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,237 @@
+name: nussknacker
+
+# todo:
```
[the remark this answers is not in the document]
Comment (coutoPL) [reply new-quickstart/docker-compose.yml]: done

File: new-quickstart/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,237 @@
...
+    restart: unless-stopped
+    environment:
+      ALLOW_ANONYMOUS_LOGIN: "yes"
+      ZOO_4LW_COMMANDS_WHITELIST: "ruok"
+    healthcheck:
+      test: ["CMD-SHELL", 'echo "ruok" | nc -w 2 -q 2 localhost 2181 | grep imok']
+      interval: 5s
+      retries: 5
+
+  kafka:
```
[the remark this answers is not in the document]
Comment (coutoPL) [reply new-quickstart/docker-compose.yml:148]: yeap, but it's by design. We want to interfere with the user's host as little as possible. The easiness of ingestion has low priority in this case.

File: new-quickstart/docker-compose.yml [hunk 5]
```diff
@@ -0,0 +1,237 @@
+name: nussknacker
```
[the remark this answers is not in the document]
Comment (coutoPL) [reply new-quickstart/docker-compose.yml:1]: ref: https://github.com/TouK/nussknacker-quickstart/pull/176#discussion_r1668409472

File: new-quickstart/quickstart-setup/data/http/generate-requests/loan.sh [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+cd "$(dirname "$0")"
+
+source ../../../scripts/utils/lib.sh
+
+ID="$(random_4digit_number)"
+AMOUNT="$(random_4digit_number)"
+REQUEST_TYPE="$(pick_randomly "loan" "mortgage" "insurance")"
+CITY="$(pick_randomly "Warszawa" "Berlin" "Gdańsk" "Kraków", "Poznań", "Praga")"
+
+echo "{\"customerId\": \"$ID\", \"requestedAmount\": $AMOUNT, \"requestType\": \"$REQUEST_TYPE\", \"location\": { \"city\": \"$CITY\", \"street\": \"\" }}"
```
Comment (raphaelsolarski) [inline new-quickstart/quickstart-setup/data/http/generate-requests/loan.sh:12]: Should `street` be empty?

Comment (coutoPL) [reply new-quickstart/quickstart-setup/data/http/generate-requests/loan.sh:12]: the scenario doesn't use this field. So I did not make an effort to generate any values ;)

**Review by lukasz-bigorajski — APPROVED** [review]
LGTM

**Review by raphaelsolarski — APPROVED** [review]

**Pull request merged**